### PR TITLE
feat: clock and pooper

### PR DESCRIPTION
**Pull request by Chiragmodi01**
- Clock and Greetings.
- Set focus, edit and delete.
- Complete focus
- Popper effect on focus complete.

### CODE REVIEW

**Review by harshitpaliwal95 — APPROVED** [review]
everything looks good

**Comment by aurobindoGupta** [comment]
netlify checks are failing, please check

File: src/screens/Main/Main.jsx [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+  const [showConfetti, setShowConfetti] = useState(false);
+
+  useEffect(() => {
+    setConfettiHeight(confettiRef.current.clientHeight);
+    setConfettiWidth(confettiRef.current.clientWidth);
+  }, [])
+  console.log(showConfetti)
 
   
   // useEffect(() => {
```
Comment (aurobindoGupta) [inline src/screens/Main/Main.jsx:22]: remove commented code

File: src/screens/Main/Main.jsx [hunk 2]
```diff
@@ -1,11 +1,22 @@
-import React, {useEffect, useState} from 'react'
+import React, {useEffect, useRef, useState} from 'react'
 import './Main.css';
 import {useBackgroundImage} from '../../hooks'
-import { Settings, Search } from '../../comps'
+import { Settings, Search, Clock } from '../../comps'
+import Confetti from 'react-confetti';
 
 function Main() {
```
Comment (aurobindoGupta) [inline src/screens/Main/Main.jsx:7]: use es6

**Review by aurobindoGupta — APPROVED** [review]
- some suggestions made plz check.
good work

**Pull request merged**